CMSgov/mint-app
# [MINT-3069] high level plans updates

# MINT-3069

## Description

- Updated high level project plan help article
- 
## How to test this change

1.  Navigate to `http://localhost:3005/help-and-knowledge/high-level-project-plan`
2. Check to see table match figma
3. Verify that the new excel file is correct

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
